service/bblfsh/bblfsh.go

```diff
@@ -42,22 +44,95 @@ func (s *Service) GetChanges(ctx context.Context, req *lookout.ChangesRequest) (
 	}
 
 	return &ChangeScanner{
-		underlying:    changes,
-		ctx:           ctx,
-		client:        s.client,
-		purgeContents: !wantContents,
+		underlying: changes,
+		BaseScanner: BaseScanner{
+			ctx:           ctx,
+			client:        s.client,
+			purgeContents: !wantContents,
+		},
 	}, nil
 }
 
-type ChangeScanner struct {
-	underlying    lookout.ChangeScanner
+func (s *Service) GetFiles(ctx context.Context, req *lookout.FilesRequest) (
+	lookout.FileScanner, error) {
+	wantContents := req.WantContents
+	if req.WantUAST {
+		req.WantContents = true
```

Comment:
Allthough this is how GetChanges was implemented - this seems a bit strange as there is no way right now for a client, to request only UASTs, isn't it?

Reply:
We have to set `req.WantContents` to get content from git service.
But we can remove it from the response when we got UAST from it.
I don't think it should be fixed in this PR. So I created an issue: https://github.com/src-d/lookout/issues/27

Reply:
oops. There is `purgeContents` already I didn't notice. Forget about it.